src/bigquery.ts

```diff
@@ -473,10 +488,18 @@ export class BigQuery extends Service {
     });
   }
 
+  get universeDomain() {
```

Comment:
Note: In the future we may want to check via Auth’s universe domain getter; which is async.